WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -840,11 +840,17 @@ private void savePostOnlineAndFinishAsync(boolean isFirstTimePublish) {
     private void onUploadSuccess(MediaModel media) {
         if (mEditorMediaUploadListener != null && media != null) {
             mEditorMediaUploadListener.onMediaUploadSucceeded(String.valueOf(media.getId()),
+                    getEditorMimeType(media.isVideo()),
                     FluxCUtils.mediaFileFromMediaModel(media));
         }
         removeMediaFromPendingList(media);
     }
 
+    private EditorFragmentAbstract.MediaType getEditorMimeType(boolean isVideo) {
```

Comment:
Seems a bit weird to keep calling `getEditorMimeType(mediaFile.getVideo()` - it seems more natural to me to have `getEditorMimeType(mediaFile)`, since that's what we're really doing (getting the media type of the media file).